site/src/components/CourseCard.vue

```diff
@@ -197,12 +197,11 @@ export default class CourseCard extends Vue {
   expanded = this.startExpanded ? this.startExpanded : false;
 
   get credMin(): string {
-    return (
-      this.course.sections[0].credMin +
-      (this.course.sections[0].credMin !== this.course.sections[0].credMax
-        ? "-" + this.course.sections[0].credMax
-        : "")
-    );
+    var min = 2147483647; // arbitrary large number
+    var max = 0;
+    this.course.sections.forEach((sec) => (min = Math.min(min, sec.credMin)));
+    this.course.sections.forEach((sec) => (max = Math.max(max, sec.credMax)));
```

Comment:
A more idiomatic way of doing this:
```typescript
let minArray = this.course.sections.map(section => section.credMin);
let maxArray = this.course.sections.map(section => section.credMax);
let min = Math.min(...minArray);
let max = Math.max(...maxArray);
```
(I haven’t actually tested this code, though…)

Reply:
That probably works (and it was actually how I first did it) but it also creates a new list/array, which I'd like to avoid.